app/client/components/holiday/holidaySelectionInfo.tsx

```diff
@@ -71,15 +70,15 @@ export const HolidaySelectionInfo = (props: HolidaySelectionInfoPros) => {
 				</div>
 				<div
 					css={{
-						[queries.maxHeight(600)]: {
+						'@media(max-height: 600px)': {
```

Comment:
This is the only instance of a vertical media query in the app, but as there's no support in Source and it's not based on any of the breakpoint definitions it's been replaced with a regular media query.